docs/src/concepts.rst

```diff
@@ -8,4 +8,6 @@
 Concepts
 ********************************************************************
 
-<placeholder text>
+Tensile is written in both Python (for library/kernel generation) and C++ (for client headers and library tests)---it is a vital project to the ROCm ecosystem, providing optimized kernels for downstream libraries such as https://github.com/rocm/rocBLAS.
```

Comment:
```suggestion
Tensile is written in both Python (for library/kernel generation) and C++ (for client headers and library tests)---it is a vital project to the ROCm ecosystem, providing optimized kernels for downstream libraries such as `rocBLAS<https://github.com/rocm/rocBLAS>`_.
```

Reply:
When you add a raw github link it makes a nice little icon beside the HTML, won't fix.